### PR TITLE
fix(android): focus on TextInputEditText view

### PR DESCRIPTION
**Pull request by garymathews**
- Fix `TiUIText` from focusing on `TextInputLayout` instead of `TiUIEditText` on Android 8.0+

##### TEST CASE
```JS
const win = Ti.UI.createWindow({
    backgroundColor: 'gray',
    layout: 'vertical'
});

const txt_a = Ti.UI.createTextField({
    hintText: 'Text Field #1',
    width: '90%'
});
const txt_b = Ti.UI.createTextField({
    hintText: 'Text Field #2',
    width: '90%'
});
const txt_c = Ti.UI.createTextField({
    hintText: 'Text Field #3',
    width: '90%'
});

txt_a.addEventListener('return', _ => {
    Ti.API.info(`'return' event fired on 'Text Field #1'`);
    txt_c.focus();
    Ti.API.info(`Set focus to 'Text Field #3'`);
});

win.add([ txt_a, txt_b, txt_c ]);
win.open();
```
1. Enter text into `Text Field #1`
2. Press `Enter`
3. `Text Field #3` should focus and allow text entry

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-27426)

### CODE REVIEW

File: android/titanium/src/java/org/appcelerator/titanium/util/TiUIHelper.java [hunk 1]
```diff
@@ -1116,20 +1116,14 @@ public static void requestSoftInputChange(KrollProxy proxy, View view)
...
+		final InputMethodManager imm =
+			(InputMethodManager) view.getContext().getSystemService(Activity.INPUT_METHOD_SERVICE);
 		if (imm != null) {
-			boolean useForce =
-				(Build.VERSION.SDK_INT <= Build.VERSION_CODES.DONUT || Build.VERSION.SDK_INT >= 8) ? true : false;
-			String model = APSAnalyticsMeta.getModel();
-			if (model.toLowerCase().startsWith("droid")) {
-				useForce = true;
-			}
+			view.requestFocus();
```
Comment (garymathews) [inline android/titanium/src/java/org/appcelerator/titanium/util/TiUIHelper.java]: Focus on `TiUIEditText` to show keyboard.

File: android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java [hunk 2]
```diff
@@ -497,12 +497,8 @@ public void focus()
 	{
 		super.focus();
 		if (tv != null) {
-			if (proxy.hasProperty(TiC.PROPERTY_EDITABLE)
-				&& !(TiConvert.toBoolean(proxy.getProperty(TiC.PROPERTY_EDITABLE)))) {
-				TiUIHelper.showSoftKeyboard(tv, false);
-			} else {
-				TiUIHelper.showSoftKeyboard(tv, true);
-			}
+			final boolean editable = proxy.getProperties().optBoolean(TiC.PROPERTY_EDITABLE, false);
```
Comment (jquick-axway) [inline android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java]: The `getProperties()` method can return `null` when an instance of this class is used by `Ti.UI.SearchBar`. In this case, an instance of this class does not have a proxy. See here...
https://github.com/appcelerator/titanium_mobile/pull/10848

**Review by jquick-axway — APPROVED** [review]
CR: Pass

**Comment by lokeshchdhry** [comment]
FR Passed.

`TextInputEditText` view gains focus & text can be added to it.

Studio Ver: 5.1.4.201909061933
SDK Ver: 8.3.0 local build
OS Ver: 10.14.5
Xcode Ver: Xcode 11.1
Appc NPM: 4.2.15
Appc CLI: 7.1.1
Daemon Ver: 1.1.3
Ti CLI Ver: 5.2.1
Alloy Ver: 1.14.1
Node Ver: 10.16.1
NPM Ver: 6.9.0
Java Ver: 10.0.2
Android Devices: ⇨ google Pixel (Android 9)
Android Emulator: ⇨ Android 8.1.0, Android 9.0, Android 4.4

**Pull request merged**